frontend/src/services/songs.ts

```diff
@@ -17,6 +17,30 @@ export async function getUserDetails() {
   }
 }
 
+export async function toggleUserChordsDisplay(isShowingChords) {
```

Comment:
Quick ask: can you type the params in these two functions? Nice for readability.

Reply:
Fixed!